app/controllers/v1/ont/runs_controller.rb

```diff
@@ -7,6 +7,24 @@ module Ont
     # See {Run} and {RunResource} for more information
     class RunsController < ApplicationController
       before_action { not_found unless Flipper.enabled?(:dpl_281_ont_create_sequencing_runs) }
+
+      # endpoint generating a sample sheet for a Ont::Run
+      def sample_sheet
+        run = ::Ont::Run.find(params[:run_id])
+        csv = run.generate_sample_sheet
+
+        send_data csv,
+                  type: 'text/csv; charset=utf-8; header=present',
+                  disposition: "attachment; filename=#{run.experiment_name}.csv"
+      end
+
+      # endpoint to retrive sample sheet setting
```

Comment:
Hopefully we won't need this anymore but best to wait on Alex.

Reply:
Ill remove for the time being. Easy enough to restore if it we do need it